docs/source/api/core/base_model.md

```diff
@@ -17,7 +17,7 @@ kernelspec:
 # API documentation for the {mod}`~virtual_rainforest.core.model` module
```

Comment:
This should also be updated to use `base_model` (once the renaming has happened)